src/Grpc.NetCore.HttpClient/GrpcClientFactory.cs

```diff
@@ -49,7 +49,10 @@ public static class GrpcClientFactory
                 handler.ClientCertificates.Add(certificate);
             }
 
-            return Cache<TClient>.Instance.Activator(new HttpClientCallInvoker(handler, new Uri(baseAddress, UriKind.RelativeOrAbsolute)));
+            var httpClient = new System.Net.Http.HttpClient(handler);
```

Comment:
Nit:

```suggestion
            var httpClient = new HttpClient(handler);
```

Reply:
The explicit namespace is there because HttpClient is the namespace. It will be renamed before release.